workshop-examples/src/main/java/_07_advanced_streaming/_01_Multi_Split.java

```diff
@@ -0,0 +1,80 @@
+///usr/bin/env jbang "$0" "$@" ; exit $?
+//DEPS io.smallrye.reactive:mutiny:2.4.0
```

Comment:
I'm wondering how you plan to keep this version in sync?

Reply:
With my fullstack shell script developer certification: https://github.com/smallrye/smallrye-mutiny/blob/main/.build/update-workshop-target-version.sh

Reply:
(and the WIP CI branch has been rebased to call it: https://github.com/jponge/smallrye-mutiny/commit/7511a87f01800d69c12bb2e77b98d5e3313150a9)